doc/pcre2pattern.3

```diff
@@ -2242,7 +2242,7 @@ package, and PCRE1 copied it from there. It found its way into Perl at release
 PCRE2 has an optimization that automatically "possessifies" certain simple
 pattern constructs. For example, the sequence A+B is treated as A++B because
 there is no point in backtracking into a sequence of A's when B must follow.
-This feature can be disabled by the PCRE2_NO_AUTOPOSSESS option, or starting
+This feature can be disabled by the PCRE2_NO_AUTO_POSSESS option, or starting
```

Comment:
This optimization can be disabled by calling the pcre2_optimize() function with a PCRE2_AUTO_POSSESS_OFF directive or by the PCRE2_NO_AUTO_POSSESS option...

There are several other references to the "legacy" options that need similar updates.

Reply:
OK, updated this and several other similar places.